src/server/controllers/resumeAndCoverLetterController.js

```diff
@@ -27,9 +27,8 @@ module.exports.updateResume = async(req, res) => {
             resume: req.file.buffer,
         }, { new: true });
         res.status(200).send(user);
-    } catch (error) {
-        res.status(400).send({ error: error.message });
-    }
+    } catch (error) {}
+
```

Comment:
Are the error messages no longer mandatory?

Reply:
they are but some of them seem redundant to me and also decrease coverage